imagersite/imager_images/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.conf import settings
+from django.utils.encoding import python_2_unicode_compatible
+
+
+PUBLISHED_CHOICES = (('private', 'private'), ('public', 'public'), ('shared', 'shared'))
+PUBLISHED_DEFAULT = PUBLISHED_CHOICES[0][1]
+
+
+@python_2_unicode_compatible
+class Photo(models.Model):
+    """Photo class."""
+
+    image = models.ImageField(upload_to='media')
+    title = models.CharField(max_length=250)
+    description = models.TextField()
+    uploaded = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    date_published = models.DateTimeField()
```

Comment:
This is making your tests fail - on Photo object creation, there is no default value provided for `date_published` which causes problems since you've also not declared that a null value is acceptable.